php/example_code/cloudfront/CreateInvalidation.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * ABOUT THIS PHP SAMPLE => This sample is part of the SDK for PHP Developer Guide topic at
+ * https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html#invalidation-specifying-objects
+ *
+ */
+
+require 'vendor/autoload.php';
+
+use Aws\CloudFront\CloudFrontClient;
+use Aws\Exception\AwsException;
+
+
+/**
+ * Invalidates an Amazon CloudFront Distribution.

```

Comment:
Should clarify that this invalidates cached objects on the specified path(s) of a distribution rather than the distribution altogether.